core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/AnyDAO.java

```diff
@@ -37,6 +39,8 @@
 
     String findKey(String name);
 
+    List<A> findByKeys(Set<String> keys);
```

Comment:
Would you mind changing this method's signature into

```java
List<A> findByKeys(List<String> keys);
```

This will ensure we keep same ordering between input and output.